u6-spas/src/views/About.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <div class="about">
+    <h1>This is an about page</h1>
+  </div>
+</template>
```

Comment:
คุณสามารถลบหน้านี้ (เส้นทางและส่วนประกอบ)
You can delete this page (route & component).